pages/ekmann.js

```diff
@@ -0,0 +1,352 @@
+          <StyledTextarea
+            type="text"
+            name="6"
+            value={formData["6"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 7 && (
+          <StyledTextarea
+            type="text"
+            name="7"
+            value={formData["7"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 8 && (
+          <StyledTextarea
+            type="text"
+            name="8"
+            value={formData["8"]}
+            onChange={handleChange}
+            placeholder="write here..."
+          />
+        )}
+        {page === 9 && (
+          <>
+            <StyledTextarea
+              type="text"
+              name="9"
+              value={formData["9"]}
+              onChange={handleChange}
+              placeholder="write here..."
+              required
+            />
+            <p>{emptyFieldsError}</p>
+          </>
+        )}
+
+        {page !== 9 && (
+          <NavButton
+            handleClick={() => {
+              setPage((currPage) => currPage + 1);
+            }}
+          >
+            next
+          </NavButton>
+        )}
+        {page === 9 && (
+          <div>
+            <NavButton handleClick={handleSubmit}>Analyze Emotion</NavButton>
+
+            <TbList onClick={() => setShowList(!showList)} />
+            {showList && (
+              <div>
+                {emotionResult && emotionResult[0].predictions && (
+                  <div>
+                    Emotion Result:
+                    <ul>
+                      {emotionResult[0].predictions.map((emotion, index) => (
+                        <li key={index}>
+                          {emotion.prediction}: {emotion.probability}
+                        </li>
+                      ))}
+                    </ul>
+                  </div>
+                )}
+              </div>
+            )}
+          </div>
+        )}
+        {predictionsState && (
+          <CircleContainer>
+            {transformedPredictions &&
+              transformedPredictions.map((emotion, index) => (
+                <Circle
+                  key={index}
+                  circleSize={emotion.probability * 500}
+                  color={getColorForEmotion(emotion.prediction)}
+                  name={emotion.prediction}
+                />
+              ))}
+          </CircleContainer>
+        )}
+      </Page>
+    </Container>
+  );
+}
+
+export default EmotionAnalysis;
+
+const CircleContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+`;
+const StyledTextarea = styled.textarea`
+  height: 150px;
+  width: 100%;
+`;
+
+const ContainerFlex = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const StyledH3 = styled.h3`
+  color: red;
+`;
```

Comment:
I think in the rest of the app there are .styled files for the styled components? Try and stay consistent here :)